samples/helloworld/src/Dockerfile

```diff
@@ -15,12 +15,12 @@
 FROM python:2-onbuild
 
 RUN mkdir -p /opt/microservices
```

Comment:
Can't we run the following command on line 25: `WORKDIR /opt/microservices` instead of mkdir. If this directory is not present, this command will create it, right? So then the next line could just be `COPY app.py .`

Reply:
```suggestion
WORKDIR /opt/microservices
```